README.md

```diff
@@ -186,7 +188,32 @@ docker run --rm --name acmedns                 \
 
 3) Copy [docker-compose.yml from the project](https://raw.githubusercontent.com/joohoi/acme-dns/master/docker-compose.yml), or create your own.
 
-4) Edit the `config/config.cfg` and `docker-compose.yml` to suit your needs, and run `docker-compose up -d`.
+4) Make sure that the user with id `1000` or group id `0` has read permissions on the configuration file in the `config` directory.
```

Comment:
What if the user with ID 1000 is already an existing user on the host system for whom it wouldn't be appropriate to grant read permissions on that config file? Is there a way to configure the container to use a different user process? Currently it's possible to do this in the existing acme-dns by setting e.g. `user: 1005:1005` in docker-compose. Will that cause any permission issues with the new container?

Reply:
Yes, changing away from 1000:1000 would cause problems. Anything "inside" the container would be owned by the wrong user ID.

If you're concerned about the host system, put your docker data under a directory only root can access. That way your host system user won't be able to access any of it. Docker runs as root and can find the correct directory for the mapping, but the host system user can't get to the subdirectory.